tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -38,6 +40,12 @@ internal class SpanMessagePackFormatter : IMessagePackFormatter<TraceChunkModel>
         private readonly byte[] _durationBytes = StringEncoding.UTF8.GetBytes("duration");
         private readonly byte[] _parentIdBytes = StringEncoding.UTF8.GetBytes("parent_id");
         private readonly byte[] _errorBytes = StringEncoding.UTF8.GetBytes("error");
+        private readonly byte[] _metaStructBytes = StringEncoding.UTF8.GetBytes("meta_struct");
+        // span links metadata
+        private readonly byte[] _spanLinkBytes = StringEncoding.UTF8.GetBytes("span_links");
+        private readonly byte[] _traceStateBytes = StringEncoding.UTF8.GetBytes("tracestate");
+        private readonly byte[] _traceFlagBytes = StringEncoding.UTF8.GetBytes("flags");
+        private readonly byte[] _tags = StringEncoding.UTF8.GetBytes("tags");
```

Comment:
Let's rename this field to `_tagsBytes` to follow the convention of all the other ones above.